src/utils/getFontsPath.js

```diff
@@ -0,0 +1,5 @@
+import path from 'path';
+
+export default (bootstrapRelPath) => {
+  return path.join(bootstrapRelPath, 'assets', 'fonts', 'bootstrap/');
```

Comment:
We need to be very careful for windows paths. There was a prior PR that dealt with a small change for windows.